packages/@yoda/util/math.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+var crypto = require('crypto')
+
+/**
+ * This function returns a floating point, the pseudorandom number is in the range [0,1),
+ * that is, from 0 (including 0) up, but not including 1 (excluding 1), then you can zoom
+ * to the desired range.
+ *
+ * This function is a temporary replacement of `random()` in standard library `math`,
+ * because that function has an unresolved issue which will return same number sequency
+ * in each new process.
+ * @returns {float} - pseudo randomized floating number within [0, 1).
```

Comment:
JavaScript has no type `float`, use `number` instead.

Reply:
Changed `float` to `number`.